docs/BR.md

```diff
@@ -444,7 +444,9 @@ The CA SHALL develop, implement, enforce, and annually update a Certification Pr
 The CA SHALL make revocation information for Subordinate Certificates and Subscriber Certificates available in accordance with this Policy.
 
 ## 2.2 Publication of information
-The CA SHALL publicly disclose its Certificate Policy and/or Certification Practice Statement through an appropriate and readily accessible online means that is available on a 24x7 basis. The CA SHALL publicly disclose its CA business practices to the extent required by the CA's selected audit scheme (see Section 8.1). The disclosures MUST include all the material required by RFC 2527 or RFC 3647, and MUST be structured in accordance with either RFC 2527 or RFC 3647. 
+The CA SHALL publicly disclose its Certificate Policy and/or Certification Practice Statement through an appropriate and readily accessible online means that is available on a 24x7 basis. The CA SHALL publicly disclose its CA business practices to the extent required by the CA's selected audit scheme (see Section 8.1).
+
+Effective as of 8 December 2017, the Certificate Policy and/or Certification Practice Statement MUST be structured in accordance with RFC 3647. Prior to 8 December 2017, the Certificate Policy and/or Certification Practice Statement MUST be structured in accordance with either RFC 2527 or RFC 3647. The Certificate Policy and/or Certification Practice Statement MUST include all material required by RFC 3647 or, if structured as such, RFC 2527.
```

Comment:
I was trying to find a clear and unambiguous way to capture the current text. Right now, it seems like a valid interpretation might be RFC 3647 format, but include all the material from RFC 2527. The first two clauses are mutually exclusive, but the third clause applies to both. I wasn't sure if that's sufficiently clear, or how to improve that so there was no misinterpretation.